xml/dm_registers.xml

```diff
@@ -571,33 +571,48 @@
         This register contains DM control and status bits that didn't easily
         fit in \RdmDmcontrol and \RdmDmstatus. All are optional.
 
-        <field name="0" bits="31:11" access="R" reset="0" />
+        If halt groups are not implemented, then \FdmDmcsTwoGroup will always
+        be 0 when \FdmDmcsTwoGrouptype is 0.
+
+        If resume groups are not implemented, then \FdmDmcsTwoGrouptype will
+        remain 0 even after 1 is written there.
+
+        The triggers referred to in \FdmDmcsTwoExttrigger might be the same,
+        or might be completely separate sets, because it is not generally
+        useful to automatically halt some cores when other cores resume
+        or vice versa.
```

Comment:
This could probably be phrased better.  I think you are saying that the external triggers available to add to halt groups may or may not be distinct from the external triggers available to add to resume groups.  In other words, external trigger 0 for halt groups might not be the same as external trigger 0 for resume groups.